content/en/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Observability Primer
+weight: 9
+---
+
+Before digging into OpenTelemetry, it is important to understand some core concepts first.
+
+## What is Observability?
+
+Observability lets us understand a system from the outside, by letting us ask questions about that system without knowing its inner workings. Furthermore, allows us to easily troubleshoot and handle novel problems (i.e. “unknown unknowns”), and helps us answer the question, "Why is this happening?" *without changing your code*.
+
+In order to be able to ask those questions of a system, code must be properly instrumented. **OpenTelemetry** is the mechanism by which application code is instrumented, to help make a system observable.
+
+## Reliability & Metrics
+
+**Telemetry** refers to data emitted from a system, about its behaviour. The data can come in the form of [Traces](#distributed-traces), [Metrics](#reliability--metrics), and [Logs](#logs).
+
+**Reliability** answers the question: “Is the service doing what users expect it to be doing?” A system could be up 100% of the time, but if, when a user clicks “Add to Cart” to add a black pair of pants to their shopping card, and instead, the system keeps adding a red pair of pants, then the system would be said to be ***un***reliable.
+
+**Metrics** are aggregations over a period of time of numeric data about your infrastructure or application. Examples include: system error rate, CPU utilization, request rate for a given service.
+
+**SLI**, or Service Level Indicator, represents an item that is being measured. Although SLI created off of [metrics](#reliability--metrics), these metrics are derived from aggragated [Trace](#distributed-traces) data. An example SLI: number of successful HTTP requests / total HTTP requests (success rate).
+
+**SLO**, or Service Level Objective, is the means by which reliability is communicated to an organization/other teams. This is accomplished by attaching one or more SLIs to business value.
+
+## Understanding Distributed Tracing
+
+To understand Distributed Tracing, let's first build on a few concepts.
+
+### Logs
```

Comment:
bunch of those concepts are already explained at other places, do we want to consolidate them?

Reply:
Yes, most definitely. In giving it a quick glance, I can see that we can probably consolidate the content from `Data sources` into `Advanced OTel Concepts` (newly added), and remove `Data sources`.